tests/test_soup.py

```diff
@@ -100,6 +100,13 @@ def fake_html_6():
     return html
 
 
+def test_find_inner_text():
+    html = """<p>&pound;600m</p>"""
+    soup = Soup(html)
+    result = soup.find("p").text
```

Comment:
This is fine! But the test case that isn't working is actually: 

```python
result = soup.text
```

Reply:
Ok, so we need to get the outer tag and then get the contents inside the tag.